### ISSUE TITLE
Pulling audio from Forvo

### ISSUE THREAD

**Issue by tryingmi** [Language-Tools/anki-hyper-tts#156]
The add-on randomly pull the audio file from forvo, not the highest rated audio file which has the most clear and best pronunciation according to the reviewers, i haven't encounter a single word that the add-on pulled the highest rated audio for it and this should be the default.

Hopefully this gets fixed soon and thanks a lot for the hard working.

**Comment by luc-languagetools** [comment]
I agree Forvo handling needs some improvements, could you share a few example words where you ran into this issue ?

**Comment by luc-vocab** [comment]
related: https://github.com/Vocab-Apps/anki-hyper-tts/issues/87
